recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
     sha256: dceb15fef556a2775121bcc695561df4ff0e09248e0ce3a2d58f5244a9f61421
 
 build:
-    number: 0
+    number: 1
```

Comment:
I'm not convinced the build number change is necessary. As I see it, it will cause many new packages and subsequent upgrades for no real reason. 

I'm leaving it for @ocefpaf to merge this is the intended way to handle this, but I'd rather just keep the build number at 0 and just add the `py36_0` packages.